### ISSUE TITLE
multiplot

### ISSUE THREAD

**Issue by addtheice** [SiegeLord/RustGnuplot#14]
Does this library allow for multiplotting?

I tried to figure it out and well...I couldn't.

**Comment by SiegeLord** [comment]
Yeah, you create multiple axes. See this code: https://github.com/SiegeLord/RustGnuplot/blob/master/examples/example1.rs#L53-L66

**Comment by addtheice** [comment]
Thank you! that solved the problem.

You might want to think about breaking up the examples in to smaller bits, showing just specific features, then naming the examples on which they where.

I just kept chewing through the docs trying to figure it out. I'm still not used to how rust organizes documentation yet.

Thanks for the assistance!
